Riot/Assets/en.lproj/Vector.strings

```diff
@@ -507,7 +508,7 @@
 "settings_term_conditions" = "Terms & Conditions";
 "settings_privacy_policy" = "Privacy Policy";
 "settings_third_party_notices" = "Third-party Notices";
-"settings_acknowledgement" = "Acknowledgement";
+"settings_acknowledgement" = "Credits";
```

Comment:
Does this change need to be reverted to?

Reply:
Nope, that's an actual change for 195396 - "Acknowledgement setting should be called 'Credits'" (bug #3 in that ticket).

Reply:
Ok then I'm confused why the 
"settings_acknowledgement" = "Credits"
change in the italian file was reverted back to:
"settings_acknowledgement" = "Acknowledgement"
and why the change from Acknowledgement -> Credits is not in all the other language files

Reply:
Good questions, I just went back and checked the Element repo as I thought that settings_acknowledgement was an Element addition that hadn't been populated through the other language files correctly, hence the other files should have been updated with the "Acknowledgement" value to match the Element default, which was what that last commit was all about.

I was actually wrong, looks like settings_acknowledgement is specific to Lingo, so it makes sense for the string value across all translations to match our default option.

I've just pushed a new commit that not only updates the string value but updates the variable name itself to reflect our usage of the string in Lingo.